packages/webapp/cypress.env.json

```diff
@@ -7,7 +7,7 @@
   "auth_password": "P4ssword!",
   "auth_user_id": "auth0|5d93178e8a968e0e14cd1573",
   "user_farm_id": "16ce7028-e42b-11e9-9717-9bd9186e761c",
-  "apiUrl": "http://localhost:5000",
+  "apiUrl": "http://localhost:5001",
```

Comment:
@MwayaB Are we sure we want to change this port number?

Cypress failed because it was not able to connect.

```
cy.request() failed trying to load:

http://localhost:5001/farm

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  > Error: connect ECONNREFUSED 127.0.0.1:5001
```

Reply:
No we don't want to change the port number. I changed the port number for local testing and forgot to discard that change when i pushed the commit